### DISCUSSION TITLE
EEG support

### DISCUSSION THREAD

**Discussion by prisciencebug** [jspsych/jsPsych#2785] [General]
Hi everyone,

I created a task using JsPsych a while ago, but my requirements have changed from running an online experiment to running an EEG experiment in the lab (our system uses the actiCHamp Plus amplifier with actiCAP caps from Brain Products).
Do you think I could implement this solution https://github.com/jodeleeuw/p300-demo to send triggers to the EEG using JsPsych? Or should I recreate the task from scratch using different software? If so, which one would you recommend? 

Any thoughts are highly appreciated!

Priscila

**Comment by luisamschmidt** [comment]
Dear Priscila,

we are currently at the exact same point as you - did you find a way to deal with sending the triggers to the EEG (Brain Vision Recorder in our case) using JsPsych? 

Would be happy for any input!

Thank you and all the best,

Luisa

**Reply by anna-lawrance** [reply]
Hi Luisa,

Did you end up switching over to PsychoPy? I am also currently working to set up an EEG experiment using jsPsych and have a Brain Vision recorder! Any input you might have would be majorly appreciated.

Thanks in advance,
Anna

**Reply by luisamschmidt** [reply]
Hi Anna,
we used jsPsych after all - my colleague and another colleague of his found a solution using a server, their approach can be found here: https://osf.io/7ka4n/ 

Hope that helps!

Best,
Luisa

**Comment by prisciencebug** [comment]
Dear Luisa,

 

We did find a way but decided to recreate the experiment in PsychoPy, as the lag between stimulus presentation and the triggers was too large (around 50 ms) with JsPsych. The solution we found is based entirely on this GitHub - jodeleeuw/p300-demo: Simple oddball detection experiment using jsPsych and EEG <https://github.com/jodeleeuw/p300-demo> .

 

Good luck!

 

Best,

Priscila
